experiments/fundamentals.py

```diff
@@ -0,0 +1,16 @@
+import os
+import pandas as pd
+
+
+def load_data(origpath, datapath, filename):
+    """
+    Given path to a specific data directory, loads in data from given filename
+    """
+    # change directory to where data is located
+    os.chdir(datapath)
```

Comment:
Try to start using the pathlib functionality. Try to get out of using os for dealing with paths.